doc/main.html

```diff
@@ -1579,6 +1593,302 @@ <h3>Complex Example</h3>
+      </p>
+      <p>
+        The GitHub release shall be named <code>rYYYY-pubState</code>, with a tag assigned to the release that matches the release name. If creating a CD, itterate the <code>pubState</code>, starting with <code>1</code>. 
+      </p>
+      <p class="example">
+        <code>r2023-fcd</code> or <code>r2023-cd2</code>
+      </p>
+      <p class="note">
+        The itteration of CD is to account for multiple possible PCD periods. 
+      </p>
+      <p>
+        Any time the <code>pubStage</code> advnaces, <code>latestEditionTag</code> shall be updated in the <code>.smpte-build.json</code> file to the value equal to the tag name of this latest release. This will trigger the tooling to create a link for <code>Redline to most recent edition</code> during any PRs and merges to the main branch. 
+      </p>
+
+      <p>
+        See <a href="#sec-sample-workflow"></a> for a sample workflow utilizing releases. 
+      </p>
+
+    </section>
+
+    <section id="sec-sample-workflow">
+      <h3>Sample Workflow </h3>
+      <p>
+        <a href="#tab-sample-workflow"></a> shows a sample workflow would be the steps for an Engineering Document going through the an initial draft or revision and the balloting process. 
+      </p>
+      <p>
+        Generally, any changes to a document shall never be made directly on the <code>Main</code> branch, which often doesn't allow the tooling to create needed redlines. Changes (regardless of the nature of the change) shall always be made on a new branch and managed via a PR (Pull Request), and then merged to <code>Main</code> after approval, as noted below at the various stages. 
+      </p>
+      <p class="note">Editors and Chairs should be aware of the general guidelines provided in the <a href="#bib-smpte-gh-om"></a>.
+      </p>
+      <table id="tab-sample-workflow" class="col-1-center">
+        <caption>Sample workflow</caption>
+        <thead>
+          <tr>
+            <th>Workflow Step</th>
+            <th>Workflow Trigger</th>
+            <th>Action(s) on document and/or GitHub repo</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>1</td>
+            <td>Project is approved</td>
+            <td>
+              <ol type="a">
+                <li>If document is an intial draft, skip to Workflow Step (2)</li>
+                <li>If document is a revision, skip to Workflow Step (3)</li>
+              </ol>
+            </td>
+          </tr>
+          <tr>
+            <td>2</td>
+            <td>Project is a document initial draft</td>
+            <td>
+              <ol type="a">
+                <li>Follow the steps in <a href="#sec-quick-start"></a> to create repo.</li>
+                <li>A new branch is created from <code>Main</code> called <code>r2023-inititial-draft</code>.</li>
```

Comment:
`Main` should be `main` everywhere

Reply:
updated in https://github.com/SMPTE/html-pub/pull/139/commits/b409f22087ea53968ede89d2f87b5c3cfaa4a969